uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeavePeriodEntitlementTest.php

```diff
@@ -23,4 +23,26 @@ public function testOnGetRemainderDoesNotAcceptPeriodIdWithoutContactId() {
     civicrm_api3('LeavePeriodEntitlement', 'getremainder', ['period_id' => 1]);
   }
 
+  /**
+   * @expectedException CiviCRM_API3_Exception
+   * @expectedExceptionMessage Unable to find a CRM_HRLeaveAndAbsences_BAO_LeavePeriodEntitlement with id 1
+   */
+  public function testGetRemainderWhenAllParametersArePassed(){
+    civicrm_api3('LeavePeriodEntitlement', 'getremainder', ['entitlement_id'=> 1, 'period_id' => 1, 'contact_id'=>1]);
+  }
+
+  /**
+   * @expectedException CiviCRM_API3_Exception
+   * @expectedExceptionMessage Unable to find a CRM_HRLeaveAndAbsences_BAO_LeavePeriodEntitlement with id 1
+   */
+  public function testGetRemainderWhenEntitlentIdAndPeriodIdArePassed(){
+    civicrm_api3('LeavePeriodEntitlement', 'getremainder', ['entitlement_id'=> 1, 'contact_id'=>1]);
```

Comment:
I don't think this is correct. The ticket on JIRA says:

> The api must received either the id of a specific entitlement, or both the contact and period id to get a list of entitlements

So, passing both the entitlement_id and the contact_id is invalid and the API should have thrown a validation error, but the error you're getting is that it couldn't find the LeavePeriodEntitlement with the given ID